FAQ.md

```diff
@@ -61,6 +61,19 @@ Another big subject. This one actually [has a devlog about it](https://dbzer0.co
 
 Connect a worker to the horde, that is all! You will generate kudos for each request you fulfil, relevant to its difficulty, and you will also generate kudos every 10 minutes your worker stays online.
 
+### How is image Kudos cost calculated?
+
+The Kudos cost reflects the amount of processing required to generate the image.
+
+* The general idea is for 50 step 512x512 image to cost 10 Kudos, 1024x1024 - 60 Kudos and 2048x2048 - 600 Kudos.
+* There is an exponential relationship between image size and kudos cost.
+* Then the step count is taken into consideration. Some samplers use more or less steps, than user actually requested, which is being taken into account too. There is a linear relationship between step count and kudos cost.

```

Comment:
I don't think this is closely related to the kudos calculation, that's more of a tip to the users :D
I think this can be added separately later if the need appears